tests/plugins/federation/test_federation_views.py

```diff
@@ -16,6 +16,14 @@ def test_federation_get_shared_events(django_app, volunteer, federation, federat
     assert federated_event.title in response.text
 
 
+def test_federation_oauth_detail_view(django_app, superuser, federation):
```

Comment:
if this should test the oauth app detail view in general i don't think this should go here. if it should test whether an oauth app is created a check for the model should be sufficient

Reply:
This checks whether the detail view works with oauth apps created in federation. Its a regression test. I moved it to the other test file though.